TileStache/Goodies/VecTiles/sort.py

```diff
@@ -76,8 +77,16 @@ def landuse(features, zoom):
     return _sort_by_area_then_id(features)
 
 
+def _place_key_desc(feature):
+    sort_key = _by_population(feature), _by_area(feature)
```

Comment:
We don't get population from WOF, but the other items in the places layer do have population, and we need to continue sorting by that key to avoid ordering regressions with existing (populated) place features.